EIPS/eip-####.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
```suggestion
---
eip: 7799
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.